app/listeners/app.init/initPassport.js

```diff
@@ -22,7 +22,7 @@ function initPassport (app) {
             .send(userData)
             .end(function (err, resp) {
             if (err) {
-                return done (err,false);
+                return done (null,false);
```

Comment:
So I have a new commit that adds an error handling component that should send a message to know what the error is. If you read the passport.js documentation at http://passportjs.org/docs/authenticate and scroll down to Verify Call back it talks about pasing an err using done here is for server exceptions where as in failed authentication should be handled using null. Unfortunately, I have yet to figure out how to access that message that is sent back. I will keep working on it but I atm I'm not sure how to do it.